src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(value) {
+      this.result += value;
+    },
+    subtract(value) {
+      this.result -= value;
```

Comment:
Better add page breaks all over the code, like here 👇 
```suggestion
  return {
    result: 0,
    
    add(value) {
      this.result += value;
    },
    
    subtract(value) {
      this.result -= value;
```